### DISCUSSION TITLE
Cloudlog First Setup Checklist

### DISCUSSION THREAD

**Discussion by magicbug** [magicbug/Cloudlog#655] [Ideas / Feature Requests]
Setup as always with any project is a bit of a pinch point and there seems to be a recurring issue of people not reading the install wiki page fully, therefore time to build a checklist for new setups.

This will show up when you hit the dashboard if you haven't carried out the tasks

- Update Country Files to populate the database
- Create a new user and delete the demo account
- Create a new station profile and set it to active

![cloudlog setup](https://user-images.githubusercontent.com/84308/95761220-e8363000-0ca3-11eb-8936-b8a1cfefa990.PNG)

**Comment by magicbug** [comment]
This issue will address #651 where user hasn't carried out the installation of country files

**Reply by AndreasK79** [reply]
@magicbug what happened to this? Did you do any more work with it?

**Reply by magicbug** [reply]
Theres a bit of code in their but needs finished

**Reply by AndreasK79** [reply]
Yes, I found it. Just a view for now I see.
